frontend_v2/src/app/components/home/home.component.html

```diff
@@ -429,8 +429,30 @@ <h3 class="fw-light center">Sponsors</h3>
         </div>
       </div>
     </section>
-    <!-- Subscribe to newsletter -->
     <section class="ev-container text-med-black ev-super-light-bg">
```

Comment:
Hey @gautamjajoo Instead of renaming `grad-container` to `grad-conatiner-testimonial` and repeating the styles of `.grad-container` at [here](https://github.com/Cloud-CV/EvalAI/pull/3527/files#diff-a5fbe0f29ae2a5d3a57f17c7e95e54dd2495fe76e40e29102b50f138f801a1c9R171) Yo can instead put an extra warpper class eg `testimonial-wrapper` here like `<section class="ev-container testimonial-wrapper text-med-black ev-super-light-bg">` and just just do 

```
.testimonial-wrapper {
    .grad-container {
      padding:0px;
    }
  }
```
under the media query of mobile devices

Reply:
Done